app/services/sufia/system_stats.rb

```diff
@@ -87,13 +87,17 @@ def validate_limit(count)
 
     def document_by_date_by_permission
       files_count = {}
-      files_count[:total] = ::GenericWork.find_by_date_created(start_date, end_date).count
-      files_count[:public] = ::GenericWork.find_by_date_created(start_date, end_date).merge(::GenericWork.where_public).count
-      files_count[:registered] = ::GenericWork.find_by_date_created(start_date, end_date).merge(::GenericWork.where_registered).count
+      files_count[:total] = query_service.find_by_date_created(start_date, end_date).count
+      files_count[:public] = query_service.find_by_date_created(start_date, end_date).merge(query_service.where_public).count
```

Comment:
Can you add a todo about consolidating the dual query_service call into another method inside query service.

```ruby
query_service.find_public_by_date_created(start_date, end_date).count
```